package-lock.json

```diff
@@ -1435,9 +1435,9 @@
       "dev": true
     },
     "node_modules/@types/node": {
-      "version": "16.18.25",
-      "resolved": "https://registry.npmjs.org/@types/node/-/node-16.18.25.tgz",
-      "integrity": "sha512-rUDO6s9Q/El1R1I21HG4qw/LstTHCPO/oQNAwI/4b2f9EWvMnqt4d3HJwPMawfZ3UvodB8516Yg+VAq54YM+eA==",
+      "version": "16.11.68",
```

Comment:
Just realised that this is going backwards (16.18.xx -> 16.11.xx) and it was down to me mis-reading the [active releases page](https://www.npmjs.com/package/@types/node?activeTab=versions). Fixed in a42ffda53dd9a27b2920577caa0981b0aaf53a15